dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -107,6 +107,7 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 			hasPageSkin: hasPageSkinConfig,
 			pageId,
 			abTests,
+			switches: { absoluteServerTimes = false },
```

Comment:
what's this?

Reply:
I've just moved it from line 137 up to here - there's no functional change. 

Here we destructure the `front` object into the properties we need and on line 137 we destructured `absoluteServerTimes` from `front.config.switches`. This change makes use do all our destructuring in one place.